server/www/index.html

```diff
@@ -141,6 +141,12 @@ <h2 ng-bind="site.name"></h2>
                 }(document, 'script'));
             }
         </script>
+        <!--
+        Intercom integration
+        -->
+        <script src="//static.intercomcdn.com/intercom.v1.js"></script>
```

Comment:
I'd like to do something similar here to the `window.ushahidi.gaEnabled` check below .. ie. don't load the JS if its not enabled. Otherwise we're loading intercom into OSS deployments where its not used.